vmdb/app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -38,6 +39,17 @@ def textual_ipaddress
     {:label => "IP Address", :value => @ems.ipaddress}
   end
 
+  def textual_memory_resources
+    {:label => "Aggregate Node Memory",
+     :value => number_to_human_size(@ems.aggregate_memory * 1.megabyte,
+                                    :precision => 0)}
+  end
+
+  def textual_cpu_cores
+    {:label => "Aggregate Node CPU Cores",
+     :value => @ems.aggregate_logical_cpus}
+  end
```

Comment:
Aggregation is done in it's own textual_group for other providers.  @dclarizio @h-kataria Thoughts here?